manifest.ts

```diff
@@ -0,0 +1,29 @@
+import {AppsBindings} from 'mattermost-redux/constants/apps';
+
+type RequestedPermissions = 'act_as_user' | 'act_as_bot'
```

Comment:
Needs to exist as constants in redux like location constants

Reply:
https://github.com/mattermost/mattermost-app-zendesk/issues/11#issuecomment-751796413